spec/requests/other_incomes_controller_spec.rb

```diff
@@ -76,7 +76,7 @@
 
         it "contains success false in the response body" do
           post_payload
-          expect(parsed_response).to match(errors: [/The property '#\/other_incomes\/1' did not contain a required property of 'source' in schema file/], success: false)
+          expect(parsed_response).to match(errors: [/The property '#\/other_incomes\/1' did not contain a required property of 'source' in schema/], success: false)
```

Comment:
@starswan It would be nice to use `expect(parsed_response[:errors])` with `include` or  `match` for consistency